app/assets/stylesheets/application.scss

```diff
@@ -30,6 +30,56 @@
  //   -ms-transform: translateY(-50%);
  //   transform: translateY(-50%);
  // }
+ .dev-tag{
+  color: black;
+  width: 30px;
+  background-color: #A80874;
+  margin: 5px 5px 5px 5px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `5px` instead of `5px 5px 5px 5px`